README.md

```diff
@@ -77,4 +81,34 @@ make test
 
 ## ROFL-8004 startup image
 
-TODO
\ No newline at end of file
+This is a container that simplifies the registration and validation of your 
+ROFL. 
+
+On the first startup it will register your ROFL as an ERC-8004 agent if it 
+doesn't exist yet and store the app ID along the agent's metadata.
+
+On every startup it will derive configured public key(s) and upsert them to 
```

Comment:
Oh it would probably be useful to add an "upsert metadata" method to the rofl appd api? Because with the current API, the containers would need to coordinate to not override each others metadata.

Reply:
Opened: https://github.com/oasisprotocol/oasis-sdk/issues/2405